saveData/save_albums.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../modules/music'
+module SaveAlbums
+  def create_album
+    puts 'Enter album genre'
+    genre = gets.chomp
+    puts 'publish date'
+    publish_date = gets.chomp
+    puts ' On Spotify?[y/n]'
+    on_spotify = gets.chomp == 'y' || false
+    @albums.push(MusicAlbum.new(publish_date, on_spotify))
+    @genres.push(Genre.new(genre))
+    puts 'New Album Created!'
+    save_albums
+    save_genres
```

Comment:
[OPTIONAL]: I suggest we run these when we exit the app, by pressing `10` in the `main.rb` file